tests/storage/sync_storage.rs

```diff
@@ -147,7 +147,9 @@ impl SyncStorage {
     }
 
     pub fn scan_lock(&self, ctx: Context, max_ts: u64) -> Result<Vec<LockInfo>> {
-        wait_op!(|cb| self.store.async_scan_lock(ctx, max_ts, cb).unwrap()).unwrap()
+        wait_op!(|cb| {
```

Comment:
Please add test to cover limit is not 0.

Reply:
Maybe The test case `test_txn_store_resolve_lock` can cover the situation which `limit` is not 0. It will call `resolve_lock_ok` and set the `limit` to `RESOLVE_LOCK_BATCH_SIZE(256)`.